modules/ocf_keyserver/files/sks-keyserver.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=OCF SKS Keyserver Instance
+Requires=network-online.target docker.service
+After=docker.service
+
+[Service]
+User=root
+# we proxy hkp from a different port so nginx can claim 11371
+ExecStart=/usr/bin/docker run --rm \
+    -v /var/lib/sks:/var/lib/sks \
+    -p 0.0.0.0:11370:11370/tcp \
```

Comment:
@cg505 was right, this needs to listen directly. nginx proxying the connection causes sks to reject peering attempts because the connections appear to come from the docker-proxy and don't match the contents of the membership file